homeassistant/components/deconz/strings.json

```diff
@@ -51,5 +51,11 @@
                 }
             }
         }
+    },
+    "device_automation": {
```

Comment:
These will need to be extended to all trigger types.

Reply:
How would I do that?

Reply:
As in, all types of triggers that you're exposing should have an entry here.

Reply:
Aah of course. Im just awaiting @emontnemery PR for the frontend

Reply:
Triggers are live in the frontend that's in dev :) conditions and actions will land tomorrow probably, reviewing actions now !

Reply:
Adopted to those changes